library/src/test/java/com/datatorrent/lib/io/fs/FileSplitterInputTest.java

```diff
@@ -110,8 +110,7 @@ protected void starting(org.junit.runner.Description description)
       fileSplitterInput.setIdempotentStorageManager(new IdempotentStorageManager.NoopIdempotentStorageManager());
 
       Attribute.AttributeMap.DefaultAttributeMap attributes = new Attribute.AttributeMap.DefaultAttributeMap();
-      attributes.put(Context.DAGContext.APPLICATION_PATH,
-          "target/" + className + "/" + methodName + "/" + Long.toHexString(System.currentTimeMillis()));
+      attributes.put(Context.DAGContext.APPLICATION_PATH, "target/" + className + "/" + methodName + "/" + Long.toHexString(System.currentTimeMillis()));
```

Comment:
That's very wierd considering no code gets into master without rat check AFAIK.
 So without these changes does checkstyle validation fails?